src/options.go

```diff
@@ -1217,7 +1217,11 @@ func parseKeyChords(str string, message string) (map[tui.Event]string, []tui.Eve
 		default:
 			runes := []rune(key)
 			if len(key) == 10 && strings.HasPrefix(lkey, "ctrl-alt-") && isAlphabet(lkey[9]) {
-				evt := tui.CtrlAltKey(rune(key[9]))
+				r := rune(key[9])
+				evt := tui.CtrlAltKey(r)
+				if r == 'h' && !util.IsWindows() {
+					evt = tui.CtrlAltBackspace.AsEvent()
+				}
```

Comment:
The problem with this approach is that `ctrl-alt-backspace` binding can inadvertently shadow `ctrl-alt-h` binding.

```sh
fzf --bind ctrl-alt-h:up+up,ctrl-alt-backspace:up
```

But I don't have a better idea at this point.